### ISSUE TITLE
Build fails in Ember 4.X when on @babel/generator >= v7.17

### ISSUE THREAD

**Issue by nlfurniss** [ember-template-imports/ember-template-imports#48]
Build fails in Ember 4.X when on @babel/generator >= v7.17

```sh
- broccoliBuilderErrorStack: Error: /home/runner/work/ember-template-imports/ember-template-imports/dummy/components/gjs-test.js: Cannot read property 'length' of undefinedTypeError: /home/runner/work/ember-template-imports/ember-template-imports/dummy/components/gjs-test.js: Cannot read property 'length' of undefined
    at memoizedBinarySearch (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@jridgewell/trace-mapping/dist/trace-mapping.umd.js:138:29)
    at traceSegmentInternal (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@jridgewell/trace-mapping/dist/trace-mapping.umd.js:506:21)
    at Object.exports.traceSegment (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@jridgewell/trace-mapping/dist/trace-mapping.umd.js:393:20)
    at originalPositionFor (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@ampproject/remapping/dist/remapping.umd.js:90:38)
    at traceMappings (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@ampproject/remapping/dist/remapping.umd.js:59:30)
    at remapping (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@ampproject/remapping/dist/remapping.umd.js:203:30)
    at mergeSourceMap (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@babel/core/lib/transformation/file/merge-map.js:22:30)
    at generateCode (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@babel/core/lib/transformation/file/generate.js:76:41)
    at run (/home/runner/work/ember-template-imports/ember-template-imports/node_modules/@babel/core/lib/transformation/index.js:55:33)
    at run.next (<anonymous>)
```

**Comment by NullVoxPopuli** [comment]
Do you still have this issue? I'm successfully using ember 4.10+ and babel 7.20+ with no issues

**Comment by tcjr** [comment]
I have this same build failure trying to use ember-template-imports with an Ember 4.11.0 project. 

Adding `"@babel/generator": "~7.16.0"` to resolutions fixed it.

**Comment by acorncom** [comment]
Looks like this can be closed? I was using this on an Ember 4.12.1 project with `@babel/generator@7.21.5` without any issues

**Comment by NullVoxPopuli** [comment]
If this issue is still present in v4, lemme know. Closing for now tho
